src/server/views/app.handlebars

```diff
@@ -69,7 +69,21 @@
             } else {
               const errorsEl = document.getElementById('document-errors');
               const errorWrapperEl = document.querySelector('.errors-wrapper');
-              errorsEl.innerHTML = res.body.errors ? JSON.stringify(res.body.errors, null, 2) : res.body.message;
+
+              function getErrorsHTMLFrom(res) {
+                if (res.body.detail) {
+                  return res.body.detail;
+                }
+
+                if (res.body.validationErrors) {
+                  const errorsSimplified = res.body.validationErrors.map(error => `${error.title} on line ${error.location.startLine}, column ${error.location.startColumn}`);
```

Comment:
- please change `on line` to `at line` to be consistent with other errors
- I also reported [this](https://github.com/asyncapi/parser-js/issues/240) bug to the parser after manually checking your PR. So as you can see `startLine` and `startColumn` can be undefined, so please make sure that when it is, you do not display an additional part of the error message about the error position

Reply:
Done!